spyder_vim/vim_widget.py

```diff
@@ -16,15 +16,16 @@
 RE_VIM_PREFIX_STR = r"^(\d*)([{prefixes}].|[^{prefixes}0123456789])(.*)$"
 RE_VIM_PREFIX = re.compile(RE_VIM_PREFIX_STR.format(prefixes=VIM_PREFIX))
 SYMBOLS_REPLACEMENT = {
-    "!": "EXCLAMATION",
-    "?": "QUESTION",
-    "<": "LESS",
-    ">": "GREATER",
-    "|": "PIPE",
-    " ": "SPACE",
-    "@": "AT",
-    "$": "DOLLAR",
-    "0": "ZERO",
+    "!":  "EXCLAMATION",
+    "?":  "QUESTION",
+    "<":  "LESS",
+    ">":  "GREATER",
+    "|":  "PIPE",
+    " ":  "SPACE",
+    "\b": "BACKSPACE",
+    "@":  "AT",
+    "$":  "DOLLAR",
+    "0":  "ZERO",
```

Comment:
The values don't need to be aligned, it's better to keep one space only after the colon. pep8 is love, pep8 is life :)